examples/machine_translation/README.md

```diff
@@ -0,0 +1,141 @@
+# This is an example to create a machine translation dataset and train a translation model.
+
+Users will use the training data in the raw file from Multi30k dataset to train a machine translation model with the character composition method.
+
+To try the example, simply run the following commands:
+
+```bash
+python train_char.py
+```
+
+For character level training, and
```

Comment:
What's the difference between "character" level vs "word" level training?

Reply:
added more explanation